cmd/examples/client.cpp

```diff
@@ -175,12 +176,18 @@ DoPublisher(const quicr::FullTrackName& full_track_name, const std::shared_ptr<q
             getline(std::cin, msg);
             SPDLOG_INFO("Send message: {0}", msg);
         }
+
         if (object_id % 10 == 0) { // Set new group
             object_id = 0;
+            subgroup_id = 0;
             group_id++;
         }
 
-        quicr::ObjectHeaders obj_headers = { group_id,       object_id++,  msg.size(),  2 /*priority*/,
+        if (object_id == 4) {
```

Comment:
Magic number?

Reply:
yes for the test app :-) we have group of 10 objects and i was thinking of creating an extra stream after 4 objects